src/styles/components/service-endpoints/styles.less

```diff
@@ -0,0 +1,12 @@
+.endpoint-download,
```

Comment:
Please avoid custom styles. We don't need this file because we can reuse existing classes from CNVS :) Please check other links for structure and behavior.

Reply:
Totally agree, but there is no other implementation or generic rule to do this as much as I searched in the project or the cnvs library.